src/instrument.cpp

```diff
@@ -6,33 +6,34 @@
 #include <arpa/inet.h>
 #include <stdlib.h>
 #include <string.h>
-#include "arch.h"
+#include <vector>
+#include "assert.h"
+#include "classfile_constants.h"
 #include "incbin.h"
+#include "log.h"
+#include "os.h"
 #include "profiler.h"
 #include "tsc.h"
 #include "vmEntry.h"
 #include "instrument.h"
 
-
 INCLUDE_HELPER_CLASS(INSTRUMENT_NAME, INSTRUMENT_CLASS, "one/profiler/Instrument")
 
+constexpr u16 MAX_CODE_LENGTH = 65534;
+
+enum {
```

Comment:
Make it named enum and let functions return this type instead of `int`.
Constants no longer need to be negative.

Reply:
bbcdd47a5df7f4273a46778ed5d908649b578964